### PR TITLE
Updates Symptom Report Page

### LINKED ISSUE #205: Update Symptom Report Text
The symptom report screen has been reverted to the old health quiz style. The buttons need to be updated to the text found in Figma. This will add an additional selection indicating no symptoms. as well as replacing some others.

These are already in the localization file as shown below:
    //symptomReport
"UX.symptomReport.heading" = "Symptom Report";
"UX.symptomReport.title" = "What symptoms do you have today?";
"UX.symptomReport.subtitle" = "Select all that apply";
"UX.symptomReport.noSymptoms" = "I don't have any symptoms today";
"UX.symptomReport.cough" = "Cough";
"UX.symptomReport.breathless" = "Breathlessness";
"UX.symptomReport.fever" = "Fever";
"UX.symptomReport.ache" = "Muscle aches";
"UX.symptomReport.loss" = "Loss of smell or taste";
"UX.symptomReport.diarrhea" = "Diarrhea";
"UX.symptomReport.nose" = "Runny nose";
"UX.symptomReport.other" = "I have symptoms that are not on the list";

### PR DESCRIPTION
**Pull request by astewart43-gtri**
closes #205

### CODE REVIEW

File: CoEpi/ui/symptomReport/SymptomReportViewController.swift [hunk 1]
```diff
@@ -54,13 +54,13 @@ class SymptomReportViewController: UIViewController, ErrorDisplayer {
 
         // TODO clarify (global) button styles
         viewModel.submitButtonEnabled
-            .map { $0 ? .systemIndigo : .lightGray }
+            .map { $0 ? .clear : .clear }
```
Comment (i-schuetz) [inline CoEpi/ui/symptomReport/SymptomReportViewController.swift:57]: To set the button's background permanently we don't need rx. Just set it directly to `.clear` in `viewDidLoad` and remove this observer.

Comment (astewart43-gtri) [reply CoEpi/ui/symptomReport/SymptomReportViewController.swift:57]: this is just a temporary change. @mellowgeek plans to use this for issue #192

File: CoEpi/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -161,6 +119,10 @@
 "UX.home.alerts2" = "    Review your potential exposures";
 "UX.home.detected" = "        new exposures detected\n\n";
 "UX.home.how" = "How is my data being used?";
+"UX.home.share" = "Share";
+"UX.home.footer.build" = "Build";
+"UX.home.footer.debug" = "Debug";
+"UX.home.footer.version" = "Version";
```
Comment (i-schuetz) [inline CoEpi/en.lproj/Localizable.strings:125]: Small note: The "UX" prefix can be dropped. It doesn't have purpose. I missed this in the original PR. You can leave the existing translations like this, but don't use it for new ones.

File: CoEpi/ui/symptomReport/SymptomReportViewController.swift (hunk 1 above)
Comment (astewart43-gtri) [reply CoEpi/ui/symptomReport/SymptomReportViewController.swift]: this was reverted as the request of @mellowgeek. His code has already been merged into develop that addresses this.

**Pull request merged**